README.md

```diff
@@ -71,10 +71,14 @@ Before downloading, the malware will create the enclosing directory if it does n
 
 If you find these files, you should delete them immediately, but consider *all* JAR files on your system compromised, and potentially all logins on your web browser as well. Passwords should be changed.
 
+You should also set up your firewall to block all connections to IPs 107.189.3.101 & 85.217.144.130 and to the website https://files-8ie.pages.dev/ip. Please note that if you are suffering a DDoS attack by this malware, you can block the User-Agent "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/112.0"
```

Comment:
Given they uploaded skyrage payload to this, blocking of ips `95.214.27.172` and `171.22.30.117` is also highly adviced as well as blocking access to `connect.skyrage.de`, `t23e7v6uz8idz87ehugwq.skyrage.de` (`files.skyrage.de`, `file.skyrage.de`, `qw3e1ee12e9hzheu9h1912hew1sh12uw9.skyrage.de`)

Reply:
I wasn't aware of it, that would probably be useful